accepted/2025/dotnet-pack-nuspec.md

```diff
@@ -0,0 +1,122 @@
+All new options and existing ones will need to be checked to see if they are applicable to the user scenario.
+This implementation does not require a new command or subcommand, making the command look cleaner and ensuring customers won't need to learn a new command.
+This is the ideal implementation.
+
+#### Example: Packing a .nuspec File with `dotnet pack`
+
+| Scenario                        | Option 1: .nuspec as Argument to `dotnet pack`                                 |
+|----------------------------------|--------------------------------------------------------------------------------|
+| Basic Usage                      | `dotnet pack MyPackage.nuspec --output ./artifacts`                             |
+| Example with Property Replacement| `dotnet pack MyPackage.nuspec --property Version=1.2.3 --property Configuration=Release --output ./artifacts` |
+| Relative Path                    | `dotnet pack ../MyPackage.nuspec --output ./artifacts`                          |
+
+### Options Relevant to .nuspec-only Support in `dotnet pack`
+
+| Option / Flag        | Description                                                        | Notes / Status for .nuspec-only support                        | Comparison (`nuget pack -h` / `dotnet pack -h`)                |
+|----------------------|--------------------------------------------------------------------|---------------------------------------------------------------|---------------------------------------------------------------|
+| `--property` / `-p:` | Set or override a property for token replacement in `.nuspec`.     | Reused; interpret for token replacement, not just MSBuild.     | `-Properties` in `nuget pack`, `--property` in `dotnet pack`   |
+| `--version`          | Specifies the version of the package.                               | Reused; overrides <version> in .nuspec if both are specified.   | `-Version` in `nuget pack`, `--version` in `dotnet pack`       |
+
+> **Note:**  
+> This table focuses on options that are new, reused with modified behavior, or require special handling for .nuspec-only support.
+> Project-specific and unrelated options (like `--no-build`) are omitted for clarity and relevance.
+> Options that are standard in the dotnet CLI, such as `--configuration` and `--output`, will be reused without modification.
+
+## Limitations
+
+- Only a subset of options are exposed as CLI flags for .nuspec-only packaging.
+- Most .nuspec metadata, including advanced fields like `<serviceable>`, must be set directly in the .nuspec file.
+- For dynamic or automated scenarios, use token replacement in the .nuspec and pass values via `--property`.
+- `--no-package-analysis`: Skips package analysis after building the package. This is not currently supported in .nuspec-only scenarios. As a result, this flag will not be available for .nuspec-only support.
```

Comment:
This is probably simple enough to add, but I'm not a fan of the mega switch to begin with, so I'm ok listed for feedback here.